test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -1,30 +1,55 @@
 #!/usr/bin/env bash
 
 function usage() {
-    echo "Usage: push_antrea.sh [--prometheus] [-h|--help]"
+    echo "Usage: push_antrea.sh [--prometheus] [-fa|--flow-agg] [-fc|--flow-collector] [-h|--help]
+    Push the latest Antrea image to all vagrant nodes and restart the Antrea daemons
+          --prometheus           Deploy Prometheus service to scrape metrics from Antrea Agents and Controllers
+          --flow-agg             Deploy Flow Aggregator service to get all flows from Flow Exporter at Antrea Agents
```

Comment:
@srikartati as per the discussions we had, I think we agreed to always enable the flow aggregator when using the FlowExporter feature. Therefore I think you can now automatically enable the aggregator when `--flow-collector` is provided, and we can get rid of `--flow-agg`.

Reply:
Yes, you are right as we do not have external flow collector support. These changes were made prior to that discussion. Removed it.